tests/layer_tests/tensorflow_tests/test_tf_SegmentSum.py

```diff
@@ -69,3 +69,54 @@ def test_segment_sum_different_types(self, params, ie_device, precision, ir_vers
         self._test(*self.create_segment_sum_net(**params),
                    ie_device, precision, ir_version, temp_dir=temp_dir,
                    use_legacy_frontend=use_legacy_frontend)
+
+class TestSegmentSumComplex(CommonTFLayerTest):
+    def _prepare_input(self, inputs_info):
+        rng = np.random.default_rng()
+        assert "data_real:0" in inputs_info
+        assert "data_imag:0" in inputs_info
+        assert "segment_ids:0" in inputs_info
+        data_real_shape = inputs_info["data_real:0"]
+        data_imag_shape = inputs_info["data_imag:0"]
+        segment_ids_shape = inputs_info["segment_ids:0"]
+        inputs_data = {}
+        inputs_data["data_real:0"] = rng.random(data_real_shape).astype(np.float32)
+        inputs_data["data_imag:0"] = rng.random(data_imag_shape).astype(np.float32)
+        inputs_data["segment_ids:0"] = rng.integers(
+            low=0, high=data_real_shape[0], size=segment_ids_shape
+        ).astype(np.int32)
+        return inputs_data
+
+    def create_segment_sum_net(self, data_shape, segment_ids_shape):
+        tf.compat.v1.reset_default_graph()
+        # Create the graph and model
+        with tf.compat.v1.Session() as sess:
+            data_real = tf.compat.v1.placeholder(tf.float32,data_shape,'data_real')
+            data_imag = tf.compat.v1.placeholder(tf.float32,data_shape,'data_imag')
+            data = tf.raw_ops.Complex(real=data_real,imag=data_imag)
+            segment_ids = tf.compat.v1.placeholder(
+                np.int32, segment_ids_shape, "segment_ids"
+            )
+            tf.raw_ops.SegmentSum(data=data, segment_ids=segment_ids)
```

Comment:
output will be of complex type. So please add extraction of real and imaginary parts of output. See tests for other operations with complex type support

Reply:
Still unresolved

Reply:
One small doubt, at last Should I `return tf_net,(real_part, imaginary_part)`

Reply:
@hegdeadithyak, check this one as reference https://github.com/openvinotoolkit/openvino/blob/master/tests/layer_tests/tensorflow_tests/test_tf_Mul.py#L248